### PR TITLE
New package: catatonit-0.1.5

### PR DESCRIPTION
**Pull request by jsbronder**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86_64)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] aarch64-musl (crossbuild)
  - [x] armv7l (crossbuild)
  - [x] armv6l-musl (crossbuild)

### CODE REVIEW

File: srcpkgs/catatonit/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+distfiles="https://github.com/openSUSE/catatonit/releases/download/v${version}/catatonit.tar.xz"
+checksum=4c45e2230235751731da3daabe130abce70b3ef37d0b1936cbc90a46b475cf08
+
+pre_configure() {
+	./autogen.sh
+}
+
+do_install() {
+	vmkdir usr/libexec/catatonit
+    vinstall catatonit 0755 usr/libexec/catatonit
```
Comment (ericonr) [inline srcpkgs/catatonit/template:20]: Funky indentation.

File: srcpkgs/catatonit/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+hostmakedepends="automake libtool"
+short_desc="Container init that is so simple it's effectively brain-dead"
+maintainer="Justin Bronder <jsbronder@cold-front.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/openSUSE/catatonit"
+distfiles="https://github.com/openSUSE/catatonit/releases/download/v${version}/catatonit.tar.xz"
+checksum=4c45e2230235751731da3daabe130abce70b3ef37d0b1936cbc90a46b475cf08
+
+pre_configure() {
+	./autogen.sh
```
Comment (ericonr) [inline srcpkgs/catatonit/template:15]: Please call `autoreconf -fi` explicitly.

File: srcpkgs/catatonit/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+do_install() {
+	vmkdir usr/libexec/catatonit
+    vinstall catatonit 0755 usr/libexec/catatonit
+
+	vmkdir usr/libexec/podman
+	ln -sf ../catatonit/catatonit "${DESTDIR}/usr/libexec/podman"
```
Comment (ericonr) [inline srcpkgs/catatonit/template:23]: Explain why this is necessary.

File: srcpkgs/catatonit/template (hunk 2 above)
Comment (jsbronder) [reply srcpkgs/catatonit/template]: done.

File: srcpkgs/catatonit/template (hunk 3 above)
Comment (jsbronder) [reply srcpkgs/catatonit/template:25]: Added a comment in the template.  For convenience:
```
    # Following suit with Fedora/Ubuntu, podman-run will default to looking
    # for catatonit here when passed the "--init" argument
```

**Comment by paper42** [comment]
Did you see #19638? I don't use this package right now, so it's fine for you to take it over.

**Comment by jsbronder** [comment]
> Did you see #19638? I don't use this package right now, so it's fine for you to take it over.

No, my apologies.  I completely failed to search and just assumed no one had submitted it.  I'll be an active user via podman for the foreseeable future at $DAY_JOB, but I don't particularly care which PR goes in.  Looks like you did build with, and then fix building with, musl.

**Comment by paper42** [comment]
> > Did you see #19638? I don't use this package right now, so it's fine for you to take it over.
> 
> No, my apologies. I completely failed to search and just assumed no one had submitted it. I'll be an active user via podman for the foreseeable future at $DAY_JOB, but I don't particularly care which PR goes in. Looks like you did build with, and then fix building with, musl.

It's totally ok, you can pull in the patch in my PR. I believe I got it from alpine. Please continue with this PR, I will close mine.

**Comment by jsbronder** [comment]
> > > Did you see #19638? I don't use this package right now, so it's fine for you to take it over.
> > 
> > 
> > No, my apologies. I completely failed to search and just assumed no one had submitted it. I'll be an active user via podman for the foreseeable future at $DAY_JOB, but I don't particularly care which PR goes in. Looks like you did build with, and then fix building with, musl.
> 
> It's totally ok, you can pull in the patch in my PR. I believe I got it from alpine. Please continue with this PR, I will close mine.

Looks like upstream already got to it.  I pulled their patch.

**Pull request merged**